source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -731,7 +731,10 @@ def merge_responses(responses):
         initial_feed_publishers = {}
         for fp in merged_response.feed_publishers:
             initial_feed_publishers[fp.id] = fp
-        merged_response.feed_publishers.extend((fp for fp in r.feed_publishers if fp.id not in initial_feed_publishers))
+
+        merged_response.feed_publishers.extend(fp for fp in r.feed_publishers
```

Comment:
will it be easy to add some tests on this?

Reply:
Unit test added